src/util.ts

```diff
@@ -131,11 +131,16 @@ function removeManifest(application: string, manifestPath: string): Promise<any>
 
 // NON-WIN32 COMMANDS //
 async function addManifestToSideloadingDirectory(application: string, manifestPath: string): Promise<void> {
-  const sideloadingDirectory = applicationProperties[application].sideloadingDirectory;
-  fs.ensureDirSync(sideloadingDirectory);
-  const sideloadingManifestPath = await getSideloadManifestPath(manifestPath, application);
-
-  fs.ensureLinkSync(manifestPath, sideloadingManifestPath);
+  try {
+    const sideloadingDirectory = applicationProperties[application].sideloadingDirectory;
+    fs.ensureDirSync(sideloadingDirectory);
+    const sideloadingManifestPath = await getSideloadManifestPath(manifestPath, application);
+    console.log(`Adding ${sideloadingManifestPath} for application ${application}`);
+    return fs.ensureLinkSync(manifestPath, sideloadingManifestPath);
+
+  } catch (err) {
+    throw new Error(`addManifestToSideloadingDirectory failed with error:\n${err}`);
```

Comment:
Suggestion for error string: `Unable to add the manifest to the sideloading directory.\n${err}`